wrappers/python/ursa_bbs_signatures/_ffi/bindings/bbs_blind_commitment.py

```diff
@@ -0,0 +1,169 @@
+import sys
+from ctypes import POINTER, byref, c_char_p, c_uint, c_ulong
```

Comment:
I think it might be good to be more explicit about the integer types, ie. `c_uint32` and `c_uint64` to avoid issues on different platforms.

Reply:
Agreed! I wasn't sure about the integer types at first and therefore chose to use `c_uint` for the time being. I think I did so for all binding functions, so I'll run through all of them and make them more explicit. Thanks! 👌